### ISSUE TITLE
Alt text not being shown in Mastodon after updating post

### ISSUE THREAD

**Issue by quazar-omega** [pixelfed/support#187]
I've updated a post of mine to add the alt text that was missing, but it doesn't seem like it is visible in Mastodon

**Comment by piebob** [comment]
i have the same issue. i wait to get to a device with a real keyboard to add alt text to my image posts. i see alt text when i hover over images in other peoples' posts, but not mine.
